examples/terraform/gce/main.tf

```diff
@@ -94,6 +94,21 @@ resource "google_compute_firewall" "internal" {
   ]
 }
 
+resource "google_compute_firewall" "nodeports" {
+  name    = "${var.cluster_name}-nodeports"
+  network = google_compute_network.network.self_link
+
+  allow {
+    protocol = "tcp"
+    ports    = ["30000-32767"]
+  }
+
+  source_ranges = [
+    "0.0.0.0/0",
+  ]
+}
```

Comment:
For the sake of consistency, can we do it in the same way as for AWS, with the `open_nodeports` variable? The same question stands for #1528, #1530, and #1533.